src/js/edu-benefits/components/SubmitMessage.jsx

```diff
@@ -63,11 +59,20 @@ export default class SubmitMessage extends React.Component {
               <span>{this.props.claimedBenefits}</span>
             </li>
             <li>
-              <b>Your relinquished benefits</b><br/>
+              <b>Benefits you gave up</b><br/>
               <span>{this.props.relinquishedBenefits}</span>
             </li>
           </ul>
         </div>
+        <h4>Documents</h4>
+        <p>This form doesn’t currently allow you to upload documents. Nothing is required right now. VA will review your application.</p>
```

Comment:
Should we say, "VA will review your application without documents" instead? Right now the thoughts seem a bit disjointed.